Tools/gsc/test/makefile

```diff
@@ -0,0 +1,138 @@
+TESTCASES?=python3 hello-world nodejs bash numpy pytorch
```

Comment:
Since we have capitalized Makefile in all other places, can you change this filename to Makefile